controllers/clustergroupupgrade_controller.go

```diff
@@ -367,6 +367,8 @@ func (r *ClusterGroupUpgradeReconciler) Reconcile(ctx context.Context, req ctrl.
 								clusterGroupUpgrade.Status.Status.CurrentBatchStartedAt.Time,
 								clusterGroupUpgrade.Status.Status.StartedAt.Time)
 
+							r.Log.Info("[Reconcile] Calculating batch timeout (minutes)", "currentBatchTimeout", fmt.Sprintf("%f", currentBatchTimeout.Minutes()))
```

Comment:
I'm not sure why this is showing up in your diff, this was already added to main in one of my recent PRs?